GiniVision/Classes/Core/GiniScreenAPICoordinator.swift

```diff
@@ -178,8 +182,49 @@ extension GiniScreenAPICoordinator: UINavigationControllerDelegate {
             }
         }
         
+        let isFromCameraToMultipage = (toVC == multiPageReviewController && fromVC == cameraViewController)
+        let isFromMultipageToCamera = (fromVC == multiPageReviewController && toVC == cameraViewController)
+        
+        if isFromCameraToMultipage || isFromMultipageToCamera {
```

Comment:
That's ugly ;(

Reply:
Then you had to see how it looked before 😅. This was the better I could come up with, do you know a better alternative?

Reply:
I was just complaining - transitions always look bad.